examples/hacker-news/__tests__/actions.test.ts

```diff
@@ -1,14 +1,12 @@
-import { createStore, applyMiddleware } from 'redux';
-
 import * as news from '../actions';
-import { middleware } from '../../../index';
 import http from '../http';
+import { initializeStore } from '../redux-store';
 
 jest.mock('../http');
 
 describe('News actions', () => {
   const dispatch = <T extends AsyncIterator<any>>(iterator: T) => {
```

Comment:
Gotta remember to clean this up. The tests are running against the async generator rather than the effects themselves, which would make this _much_ easier.